packages/ember-glimmer/tests/utils/abstract-test-case.js

```diff
@@ -148,8 +148,16 @@ export class RenderingTest extends TestCase {
     run(callback);
   }
 
-  registerHelper(name, func) {
-    this.owner.register(`helper:${name}`, helper(func));
+  registerHelper(name, funcOrClassBody) {
+    let type = typeof funcOrClassBody;
+
+    if (type === 'function') {
```

Comment:
This is an internal abstraction for the test. [This](https://github.com/emberjs/ember.js/pull/12917/files#diff-b694f4d428d9270e19cfad2f9eca7815R6) and [this](https://github.com/emberjs/ember.js/pull/12917/files#diff-b694f4d428d9270e19cfad2f9eca7815R14) is how that is being used in practice. The reason is that we cannot expose `Helper` and `helper` to the test file (because they are different between `ember-htmlbars` and `ember-glimmer`).